modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,25 +1,30 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+Perhaps your organization requires someone with the proper authority to deploy Mule applications once the design phase has been completed.
```

Comment:
The way I understand it, there are different personas using the product. One persona is a designer type that designs apps all day but is not the one who deploys them into production. In this case, one persona hands off the app to another. The way the PM describes it, this is the case where you need to give the app to someone who can deploy it. The other file covers the other use case. I can reword it slightly, but the reason is not a limit with the tool, it's a limit by persona.
Here's a rewrite with your feedback in mind :
After designing a Mule application in Anypoint Flow Designer, someone other than yourself may need to deploy it. You must first use Flow Designer to export the project as a .jar file, in order to share it with someone with deployment credentials so that they can deploy the app using Anypoint Runtime Manager.

Reply:
Fixed.